packages/unplugin-typia/src/core/typia.ts

```diff
@@ -110,6 +114,29 @@ async function getProgramAndSource(
 	);
 	const host = ts.createCompilerHost(compilerOptions);
 
+	host.resolveModuleNameLiterals = (moduleLiterals, containingFile) => {
+		return moduleLiterals.map((lit) => {
+			let alias;
+			// eslint-disable-next-line no-cond-assign
+			if (aliases && !lit.text.startsWith('./') && (alias = findMatchingAlias(lit.text, aliases))) {
+				let extension;
+				if (lit.text.endsWith('.d.ts')) {
```

Comment:
Do we really need for `.d.ts` statement?

Reply:
That's because `path.extname` will only parse it as `.ts`. There could probably be a better solution, do you have ideas?

Reply:
Got it. That looks good for me now